downstream/assemblies/eda/assembly-eda-set-up-rhaap-credential.adoc

```diff
@@ -2,6 +2,16 @@
 
 = {PlatformName} credential
 
-The {PlatformName} credential type can connect to {ControllerName} through the use of an {ControllerName} URL and a username and password. After you have created this credential type, it can be attached to a rulebook and used to run rulebook activations.  
+When {EDAcontroller} is deployed on {PlatformNameShort} {PlatformVers}, you can create a {PlatformName} credential to connect to {ControllerName} through the use of an {ControllerName} URL and a username and password. After it has been created, you can attach the {PlatformName} credential to a rulebook and use it to run rulebook activations. These credentials provide a simple way to configure communication between {ControllerName} and {EDAcontroller}, enabling your rulebook activations to launch job templates.
 
+[NOTE]
+====
+If you deployed {EDAcontroller} with {PlatformNameShort} 2.4, you probably used controller tokens to connect {ControllerName} and {EDAcontroller}. These controller tokens have been deprecated in {PlatformNameShort} {PlatformVers}. 
```

Comment:
Maybe this is overkill, but what do you think about adding: 
```suggestion
If you deployed {EDAcontroller} with {PlatformNameShort} 2.4, you probably used controller tokens to connect {ControllerName} and {EDAcontroller}. These controller tokens have been deprecated in {PlatformNameShort} {PlatformVers} and must be deleted.
```

Reply:
I had that sentence added to the note before, but opted to move it to the end of the overview section after speaking to SMEs.